iOS_SDK/OneSignalSDK/Source/NSURL+OneSignal.m

```diff
@@ -37,4 +38,28 @@ - (NSString *)valueFromQueryParameter:(NSString *)parameter {
     
     return nil;
 }
+
+- (NSString*)supportedFileExtension {
+    NSURLComponents *components = [NSURLComponents componentsWithURL:self resolvingAgainstBaseURL:false];
+    
+    for(int i = (int)components.queryItems.count-1; i > -1; i--) {
```

Comment:
is the cast needed? 
can you reformat the code, please 😁 for (int i = (int)components.queryItems.count - 1; i > -1; i--)

Reply:
It is using the reverse enumerator now so it should be much cleaner

Reply:
haha yeah I mean components.queryItems.count - 1 <-- space between that

Reply:
There were 2 backwards loops and I only updated one. Now they are both using the reverseObjectEnumerator!